docs/code-reviews.md

```diff
@@ -1,4 +1,67 @@
-!!! note "These documents are a work-in-progress"
-    This section needs to be written! A "code review" is an important quality assurance practice. Every time you make a Pull Request in Github, you should ask a collaborator to do a review. [This video](https://github.com/features/code-review/) is a good starting point.
+## What is a code review?
 
----8<-- 'includes/glossary.md'
+A code review is a critical quality assurance practice. It involves systematically
+checking a fellow programmer's (or researcher's) code for errors or possible 
+improvements. 
+
+It is an everyday activity in the software development world and improves
+both quality and speed in programming. 
+
+## How to ask for a review
+
+Github has a feature built into the workflow that allows easy code review with collaborators, and we would recommend 
```

Comment:
Add a link here to the review feature page on github docs? https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews

Reply:
Good idea